packages/@react-spectrum/list/src/ListViewItem.tsx

```diff
@@ -126,10 +126,11 @@ export function ListViewItem<T>(props: ListViewItemProps<T>) {
   let showCheckbox = state.selectionManager.selectionMode !== 'none' && state.selectionManager.selectionBehavior === 'toggle';
   let {visuallyHiddenProps} = useVisuallyHidden();
 
+  let dropProps = isDroppable ? droppableItem?.dropProps : {'aria-hidden': droppableItem?.dropProps['aria-hidden']};
```

Comment:
Disabled items have `isDroppable` resolved to false so we were omitting all drop props previously. However, we do still want `aria-hidden` to be applied to the disabled row when a drag session is underway so that those rows are skipped